workers/default/arachne/arachne_db_publish.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import os
+import glob
+
+from utils import mysql
+
+log = logging.getLogger(__name__)
+
+
+PDF_PATH = 'aronscans/download-book'
+METS_PATH = 'S-Arachne/MetsDocuments'
+PTIF_PATH = 'S-Arachne/arachne4scans/arachne4webimages/bookscans'
+ARCHIVE_PATH = 'historical-books-archive/DAI'
+
+
+def add_book(book, object_id):
+    """Write book data to Arachne-Database."""
+    book_metadata = book.metadata.to_dict()
+    author = (f"{book_metadata['creator']['lastname']}, "
+              f"{book_metadata['creator']['firstname']}")
+    page_count = len(glob.glob(book.get_representation_dir('origin') + '/*'))
+
+    book_query = ("INSERT INTO arachne.buch"
+                  "(DatensatzGruppeBuch, BearbeiterBuch, creatienDateTime, "
+                  "creation, lastModified, Verzeichnis, origFile, "
+                  "KurzbeschreibungBuch, BuchAuthor, BuchTitel, BuchJahr, "
+                  "BuchOrt, BuchSeiten, Materialbeschreibung, "
+                  "ZusaetzlicheMasze, BuchMaszeBemerk, hasOcrText, "
+                  "hasMarcData) "
+                  "VALUES(%s, %s, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP, "
+                  "CURRENT_TIMESTAMP, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s,"
+                  " %s, %s, %s)")
+    book_args = ('Arachne', 'cilantro', object_id, '',
```

Comment:
Spricht etwas dagegen, statt "cilantro" den Namen des Cilantro-Users in das Feld `BearbeiterBuch` zu schreiben? Vielleicht mit dem Zusatz "(via iDAI.workbench)"?

Reply:
Gute Idee, ist so eingebaut.